keywording/text.xml

```diff
@@ -243,6 +252,29 @@ to do this.
 <title>Moving from <c>~arch</c> to <c>arch</c></title>
 <body>
 
+<p>
+If a package has stable keywords, maintainers should regularly (subject to the
+rules below) file stabilization bugs for their packages, ideally approximately
+every 30 days after a new version is added. If a bug report for stabilization
+is not filed by the maintainer, the maintainer should respond with an
+acknowledgement ("ACK") if the ebuild is ready, and a negative
+acknowledgment ("NAK") if not.
```

Comment:
> acknowledgement is preferred in British English, acknowledgment in American English.

which one was the preferred one for devmanual? :)

```suggestion
acknowledgement ("ACK") if the ebuild is ready, or a negative
acknowledgement ("NAK") if not.
```

Reply:
> > acknowledgement is preferred in British English, acknowledgment in American English.
> 
> which one was the preferred one for devmanual? :)

British English: https://devmanual.gentoo.org/appendices/contributing/index.html#style-guidelines

Reply:
Thanks!